src/tests/checkers/basic_test.go

```diff
@@ -2174,3 +2174,40 @@ function f() {
 	}
 	test.RunAndMatch()
 }
+
+func TestArrayKeyExistCallWithObject(t *testing.T) {
+	test := linttest.NewSuite(t)
+	test.AddFile(`<?php
+function array_key_exists($a, $b): bool { return true; }
+
+class Foo {}
+
+function returnObject(): Foo {
+	return new Foo;
+}
+
+function returnObjectAndNull(): ?Foo {
+	if (1) {
+		return null;
+	}
+	return new Foo;
+}
+
+function f() {
+    $foo = new Foo;
+	$arr = ["a" => 100];
```

Comment:
Minor: this line looks to be aligned inconsistently with other lines.

Reply:
Fixed.